### ISSUE TITLE
qt5 depends

### ISSUE THREAD

**Issue by ustsv** [Alexpux/MINGW-packages#3038]
qt5 depends on pcre2 (not only pcre and/or?) 
Qt5Core.dll import libpcre2-16-0.dll

**Comment by mati865** [comment]
duplicate of https://github.com/Alexpux/MINGW-packages/issues/3036
